packages/three-vrm-core/src/humanoid/VRMHumanoidRig.ts

```diff
@@ -130,7 +131,8 @@ export class VRMHumanoidRig extends VRMRig {
 
         // Move the mass center of the VRM
         if (boneName === 'hips') {
-          const boneWorldPosition = rigBoneNode.getWorldPosition(new THREE.Vector3());
+          const boneWorldPosition = rigBoneNode.getWorldPosition(_boneWorldPos);
+          boneNode.parent!.matrixWorld.multiplyMatrices(boneNode.parent!.parent!.matrixWorld, boneNode.parent!.matrix);
```

Comment:
I believe `updateMatrixWorld(false, false)` says the same thing.

also, why we don't need to do `updateMatrixWorld(true, false)` here?

Reply:
Yes updateWorldMatrix(false,false) does almost the same thing but it add an extra updateMatrix that isn't necessary here.

I don't remember exactly why but it would say that the parent matrixWorld is already up to date and correct.
calling the method would trigger some unnecessary matrix updates and impact the perf.

Reply:
> calling the method would trigger some unnecessary matrix updates and impact the perf.

yes indeed but we want to make sure that we update all the necessary matrices here.

I just have built a breadboard of the specific case and now I'm 95% sure it has to be `updateWorldMatrix(true, false)`:

https://glitch.com/edit/#!/early-wooded-socks

Reply:
Mmmmm something feels wrong for me in this example. 
I'll check again by the end of the day if you don't mind waiting. 
I'll explain fully why I think it's not necessary to use `updateWorldMatrix(true, false)` in a normal way of updating a VRM and then you will be able to tell me if you agree or not. 🙇

Reply:
take your time, I'll wait for you :+1:

In my understanding, world matrices of objects in a scene usually update upon `renderer.render`, but we need the world matrix before the render call, so we need to update the ancestors of the `boneNode`.

Reply:
Ok I think you're right!
In my test case I was moving vrm.scene, I thought it was the only "correct" way to move a skinnedMesh

I tried your example using vrm.scene.position.x = 1.0; instead of moving the hips and my solution works.
But I didn't consider that hips could move too and in that scenario my solution breaks.

I'm also still a bit confused about the difference between hipsNormalized and hipsRaw and some other aspects of VRM so I won't dig deeper for now.

I updated the PR accordingly 👍

Reply:
ok! @ https://github.com/pixiv/three-vrm/pull/1057/commits/e0a1535c53493f4a582c1519a5a9db021255fb7e